R/router.R

```diff
@@ -160,11 +173,79 @@ create_router_callback <- function(root, routes) {
   }
 }
 
+#' Create router UI
+#'
+#' Creates router UI in Shiny applications.
+#'
+#' @param default Main route to which all invalid routes should redirect.
+#' @param ... All other routes defined with shiny.router::route function.
+#' @param page_404 Styling of page when wrong bookmark is open. See \link{page404}.
+#' @param env Environment (only for advanced usage).
```

Comment:
Suggestion:
I assume that this needed for being able to define a router inside a module, if yes maybe we could document that?

```suggestion
#' @param env Environment (only for advanced usage), makes it possible to use shiny.router inside shiny modules.
```